src/components/listing-search-page/SearchResults/listing-search-cards.style.js

```diff
@@ -37,14 +39,58 @@ export const SearchButton = styled.button`
 
 export const CardsContainer = styled.div`
   max-height: 40rem;
-  overflow: scroll;
+  overflow-y: scroll;
+`;
+
+export const SorryMessage = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 5rem 3rem;
+`;
+
+export const Message = styled.p`
+  text-align: center;
+  width: 90%;
+  color: ${colors.dustyGray};
+  font-size: 1.2rem;
+`;
+
+export const SuggestedListing = styled.p`
+  font-size: 1.3rem;
+  margin-left: 2rem;
+`;
+
+export const ResultCount = styled.div`
+  padding-left: 2rem;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const ResultNumber = styled.p`
+  font-weight: 600;
+  padding-right: 0.5rem;
+`;
+
+export const ResultText = styled.p`
+  font-weight: 600;
+  color: ${colors.dustyGray};
```

Comment:
```suggestion
export const Text = styled.p`
....
```
Please create one shared Text then extend your components